server/src/main/java/org/elasticsearch/tasks/TaskInfo.java

```diff
@@ -196,7 +196,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         }
         builder.field("running_time_in_nanos", runningTimeNanos);
         builder.field("cancellable", cancellable);
-        if (parentTaskId.isSet()) {
+        if (parentTaskId.isSet() && parentTaskId.getNodeId().equals("cluster") == false) {
```

Comment:
Let us add a todo here, to ensure we revisit this. I think this means that `cluster:` parent ids no longer appear in the `_tasks` output, which could harm diagnostics/troubleshooting.